### PR TITLE
docs(forms): Forms docs update

### PR DESCRIPTION
**Pull request by massao**
Move examples to separate files for `Form` and `FormControl` documentation

### CODE REVIEW

File: packages/components/forms/src/Form/README.mdx [hunk 1]
```diff
@@ -26,68 +21,34 @@ Go ahead and add some fields into your form:
...
 
+## Import
+
+```js static=true
+import { Form } from '@contentful/f36-components';
+// or
+import { Form } from '@contentful/f36-forms';
+```
+
 ## Code examples
```
Comment (suevalov) [inline packages/components/forms/src/Form/README.mdx:32]:
```suggestion
## Examples
```

File: packages/components/forms/src/Form/README.mdx [hunk 2]
```diff
@@ -26,68 +21,34 @@ Go ahead and add some fields into your form:
...
 
 ### Basic example
```
Comment (suevalov) [inline packages/components/forms/src/Form/README.mdx:34]:
```suggestion
### Basic
```

File: packages/components/forms/src/Form/README.mdx [hunk 3]
```diff
@@ -26,68 +21,34 @@ Go ahead and add some fields into your form:
...
+
 ```
 
+## Content guidelines
+
+- The `Form` component should wrap form elements into `<form />`
+- Provide an `onSubmit` handler for form validation and submission
+- The state of the validation of the form should be manipulated from outside
+
 ### Integrations
```
Comment (suevalov) [inline packages/components/forms/src/Form/README.mdx:52]:
```suggestion
## Integrations
```

File: packages/components/forms/src/FormControl/README.mdx [hunk 4]
```diff
@@ -31,87 +39,38 @@ To uphold a consistent look and experience for your application, we recommend fo
 
 ### Basic example
```
Comment (suevalov) [inline packages/components/forms/src/FormControl/README.mdx:40]: please rename `Usage examples` to `Examples`, and `Basic example` to `Basic`

File: packages/components/forms/src/FormControl/README.mdx [hunk 5]
```diff
@@ -31,87 +39,38 @@ To uphold a consistent look and experience for your application, we recommend fo
...
-<FormControl isRequired>
-  <FormControl.Label>Name</FormControl.Label>
-  <TextInput />
-  <FormControl.HelpText>Please enter your first name</FormControl.HelpText>
-</FormControl>
+```jsx file=../../examples/FormControlExample.tsx
+
 ```
 
 ### Example with invalid input
```
Comment (suevalov) [inline packages/components/forms/src/FormControl/README.mdx:46]: Just drop all `Example...` from headings. They are all under Examples group.

File: packages/components/forms/src/FormControl/README.mdx [hunk 6]
```diff
@@ -31,87 +39,38 @@ To uphold a consistent look and experience for your application, we recommend fo
...
-      Name should be no longer than 8 characters
-    </FormControl.HelpText>
-    <FormControl.Counter />
-  </Flex>
-</FormControl>
+```jsx file=../../examples/FormControlCharacterCountExample.tsx
+
 ```
 
 ### Props
```
Comment (suevalov) [inline packages/components/forms/src/FormControl/README.mdx:76]:
```suggestion
## Props (API reference)
```

**Review by thebesson — APPROVED** [review]

**Review by suevalov — APPROVED** [review]

**Pull request merged**